src/styles/catalog.scss

```
@@ -0,0 +1,132 @@
+.catalog {
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  grid-template-columns: repeat(4, 200px);
+  align-items: center;
+  justify-content: center;
+  gap: 46px 48px;
+  padding: 50px 40px;
+
+  @media (max-width: 487px) {
+    grid-template-columns: repeat(1, 200px);
+  }
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+
+  .card {
+    width: 200px;
+    padding: 32px 16px 16px;
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+    display: flex;
+    flex-direction: column;
+
+    .card__img {
+      width: 160px;
+      height: 134px;
+      margin-left: 3px;
+    }
+
+    .card__title {
+      margin: 40px 0 -6px;
+      font-weight: 500;
+      font-size: 12px;
+    }
+
+    .card__code {
+      color: $secondary;
+      font-size: 10px;
+      line-height: 14px;
+      margin-bottom: 7px;
+      font-weight: 300;
+    }
+
+    .card__group-stars {
+      display: flex;
+      align-items: center;
+      justify-content: space-between;
+
+      .stars {
+        display: flex;
+        align-items: center;
+
+        .star {
+          display: block;
+          width: 16px;
+          height: 16px;
+          background-image: url('../images/star.svg');
+          background-repeat: no-repeat;
+          background-position: center;
+          margin-left: 4px;
+
+          &:first-child {
+            margin-left: 0;
+          }
+        }
+      }
+
+      .stars--4 .star:nth-child(-n + 4) {
```

```suggestion
      .stars--4 .star:nth-child(-n + 4),
      ...,
      .stars--1 .star:nth-child(-n + 1), {
```
All modifiers should work properly